source/extensions/filters/common/rbac/matchers/upstream_ip_port.cc

```diff
@@ -16,8 +16,13 @@ using namespace Filters::Common::RBAC;
 
 UpstreamIpPortMatcher::UpstreamIpPortMatcher(
     const envoy::extensions::rbac::matchers::upstream_ip_port::v3::UpstreamIpPortMatcher& proto) {
+  if (!proto.has_upstream_ip() && !proto.has_upstream_port_range()) {
+    throw EnvoyException("Invalid UpstreamIpPortMatcher configuration - missing `upstream_ip` and "
```

Comment:
```suggestion
    throw EnvoyException("Invalid UpstreamIpPortMatcher configuration - missing `upstream_ip` and/or "
```

Reply:
Also, update the docs to make it more clear that one or both are required (I already mentioned this), thanks.